src/frontends/onnx/frontend/src/op/greater_or_equal.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/greater_or_equal.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "default_opset.hpp"
+
+OPENVINO_SUPPRESS_DEPRECATED_START
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector greater_or_equal(const Node& node) {
+    const auto A = node.get_ng_inputs().at(0);
+    const auto B = node.get_ng_inputs().at(1);
+
+    const auto C = std::make_shared<default_opset::GreaterEqual>(A, B);
+
+    return {C};
+}
+}  // namespace set_1
+
+namespace set_12 {
```

Comment:
As I can see in docs - it has two versions 12 and 16. In such case initial version 12 should be in set_1, and newer version 16 in set_16. Could you align, please?